### ISSUE TITLE
Consistent language

### ISSUE THREAD

**Issue by popzip** [marklogic/marklogic-data-hub#1173]
Harmonize (ha, ha) language related to Operational Data Hub, Data Integration, DHF. 
Recent diagrams introduce Curation - how does this differ from Harmonization?
Use of Staging/final (database names in DHF) vs "As-Is"/Raw/Source, Target/Canonical/Curated
Access/Serve

feel free to add others here.
largely between existing DHF documentation and the ODH diagram, as well as evolution of terminology and usage.

**Comment by kcoleman-marklogic** [comment]
See also DHFPROD-1048. Related, if not necessarily a duplicate.

**Comment by wooldridge** [comment]
Use of entity/model. We define an entity (e.g., Product) under the Entities view. That entity is labeled as the "Model" in the mapping UI.

**Comment by wooldridge** [comment]
Use of property/attribute. Not sure if this surfaces currently in the app or the documentation, but I hear these terms used interchangeably in our discussions.

**Comment by kcoleman-marklogic** [comment]
Those are good ones, Mike.

There's a relevant set of T&D's here: http://docs.marklogic.com/guide/entity-services/intro#id_20667. We can adjust as needed, but DHF is not a special snowflake and should align with ES for shared concepts, IMO.

>Use of property/attribute.

We had the property vs attribute discussion when we implemented ES. One can go either way, but we chose property for ES, so let's use it in DHF.

The ES guide tries to be very explicit: entity type, entity instance, entity property. Almost never will you find just "entity", "property", or "attribute".  The unadorned terms are too ambiguous, at least in the docs. Maybe not so true in the UI.

>Use of entity/model. 

DHF asserts one type per model, which is more restrictive than what ES conceptualizes and supports. That aggravates the ambiguity.

What QS calls the model is an entity type definition. It doesn't really become a model until it is persisted in ML and gets fluffed up with "facts" (triples). 

When we worked on ES, @jmakeig was quite adamant that the descriptor is NOT the model. The ES guide distinguishes between creating a model descriptor (the entity type definition) and creating the model.  

I think this distinction matters to DHF as whole, but I'm not sure if it is important for QS. The realization of the model in ML is pretty obscured from the user in QS. When does the model realization happen?
